assets/js/18f-guide.js

```diff
@@ -0,0 +1,11 @@
+$(function() {
+  $('.sidebar-nav').each(function() {
+    accordion($(this));
+  });
+});
+
+// http://bryanbraun.github.io/anchorjs/
+anchors.options = {
+  visible: 'touch'
+};
+anchors.add('h3,h4,h5');
```

Comment:
That makes sense. The "right thing" is to make sure the page title is the only `<h2>`. Let's keep `h2` of of this, then.